<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QVInfoDialog</class>
 <widget class="QDialog" name="QVInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>263</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>File Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>337</width>
        <height>982</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <property name="rowWrapPolicy">
        <enum>QFormLayout::RowWrapPolicy::DontWrapRows</enum>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="nameLabel">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="text">
          <string>error</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Type:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="typeLabel">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="text">
          <string>error</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Size:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="sizeLabel">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="text">
          <string>error</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label6">
         <property name="text">
          <string>Location:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="locationLabel">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="text">
          <string>error</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Modified:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="modifiedLabel">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="text">
          <string>error</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label4">
         <property name="text">
          <string>Dimensions:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="dimensionsLabel">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="text">
          <string>error</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label5">
         <property name="text">
          <string>Aspect Ratio:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="ratioLabel">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="text">
          <string>error</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="framesLabel2">
         <property name="text">
          <string>Frames:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="framesLabel">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="text">
          <string>error</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Camera make:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="label_camera_make">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Camera model:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="label_camera_model">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Software:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="label_software">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Bits per sample:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="label_bits_per_sample">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Image description:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="label_description">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Image copyright:</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="label_copyright">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Image date/time:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLabel" name="l_image_time">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Original date/time:</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QLabel" name="label_original_time">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Digitize date/time:</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QLabel" name="label_digitize_time">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Subsecond time:</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QLabel" name="l_subsecond_time">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Exposure time:</string>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="QLabel" name="label_exposure_time">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="19" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>F-stop:</string>
         </property>
        </widget>
       </item>
       <item row="19" column="1">
        <widget class="QLabel" name="label_F_stop">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="20" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Exposure program:</string>
         </property>
        </widget>
       </item>
       <item row="20" column="1">
        <widget class="QLabel" name="label_exposure_program">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="21" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>ISO speed:</string>
         </property>
        </widget>
       </item>
       <item row="21" column="1">
        <widget class="QLabel" name="label_ISO">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="22" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Subject distance:</string>
         </property>
        </widget>
       </item>
       <item row="22" column="1">
        <widget class="QLabel" name="label_subject_distance">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="23" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Exposure bias:</string>
         </property>
        </widget>
       </item>
       <item row="23" column="1">
        <widget class="QLabel" name="label_exposure_bias">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="24" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Flash used:</string>
         </property>
        </widget>
       </item>
       <item row="24" column="1">
        <widget class="QLabel" name="label_flash_used">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="25" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Flash returned light:</string>
         </property>
        </widget>
       </item>
       <item row="25" column="1">
        <widget class="QLabel" name="label_flash_returned_light">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="26" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Flash mode:</string>
         </property>
        </widget>
       </item>
       <item row="26" column="1">
        <widget class="QLabel" name="label_flash_mode">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="27" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Metering mode:</string>
         </property>
        </widget>
       </item>
       <item row="27" column="1">
        <widget class="QLabel" name="l_metering_mode">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="28" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Lens focal length:</string>
         </property>
        </widget>
       </item>
       <item row="28" column="1">
        <widget class="QLabel" name="l_lens_focal_length">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="29" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>35mm focal length:</string>
         </property>
        </widget>
       </item>
       <item row="29" column="1">
        <widget class="QLabel" name="l_35mm_focal_length">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="30" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>GPS Latitude:</string>
         </property>
        </widget>
       </item>
       <item row="30" column="1">
        <widget class="QLabel" name="label_gps_latitude">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="31" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>GPS Longitude:</string>
         </property>
        </widget>
       </item>
       <item row="31" column="1">
        <widget class="QLabel" name="label_gps_longtitude">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="32" column="0">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>GPS Altitude:</string>
         </property>
        </widget>
       </item>
       <item row="32" column="1">
        <widget class="QLabel" name="label_gps_altitude">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="33" column="0">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>GPS Precision (DOP):</string>
         </property>
        </widget>
       </item>
       <item row="33" column="1">
        <widget class="QLabel" name="label_gps_precision">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="34" column="0">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>Lens min focal length:</string>
         </property>
        </widget>
       </item>
       <item row="34" column="1">
        <widget class="QLabel" name="l_lens_min_focal_length">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="35" column="0">
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>Lens max focal length:</string>
         </property>
        </widget>
       </item>
       <item row="35" column="1">
        <widget class="QLabel" name="l_lens_max_focal_length">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="36" column="0">
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>Lens f-stop min:</string>
         </property>
        </widget>
       </item>
       <item row="36" column="1">
        <widget class="QLabel" name="l_lens_F_min">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="37" column="0">
        <widget class="QLabel" name="label_34">
         <property name="text">
          <string>Lens f-stop max:</string>
         </property>
        </widget>
       </item>
       <item row="37" column="1">
        <widget class="QLabel" name="l_lens_F_max">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="38" column="0">
        <widget class="QLabel" name="label_35">
         <property name="text">
          <string>Lens make:</string>
         </property>
        </widget>
       </item>
       <item row="38" column="1">
        <widget class="QLabel" name="l_lens_make">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="39" column="0">
        <widget class="QLabel" name="label_36">
         <property name="text">
          <string>Lens model:</string>
         </property>
        </widget>
       </item>
       <item row="39" column="1">
        <widget class="QLabel" name="l_lens_model">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
